Resources/config/services.yml

```diff
@@ -1,27 +1,15 @@
 services:
-    Geocoder\Dumper\GeoArray:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\GeoJson:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Gpx:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Kml:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Wkb:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Wkt:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
+    _instanceof:
+        Geocoder\Dumper\Dumper:
+            tags: ['bazinga_geocoder.dumper']
+            public: true
+
+    Geocoder\Dumper\GeoArray: ~
+    Geocoder\Dumper\GeoJson: ~
```

Comment:
Are these needed?

Reply:
Dumpers would not be registered otherwise :thinking:

Reply:
Oh, sorry. It is too early in the morning. My two comments are invalid.